ansible/server-state-playbooks/www/www.yml

```diff
@@ -58,6 +84,7 @@
       jekyll_build_git_repo: "https://github.com/openmicroscopy/www.openmicroscopy.org"
       jekyll_build_force_rebuild: True
       jekyll_build_config: ['_config.yml', '_prod.yml']
+      jekyll_build_name: www.openmicroscopy.org
```

Comment:
One more commit commit to reflect this change in https://github.com/openmicroscopy/infrastructure/blob/master/ansible/server-state-playbooks/www/www.yml#L212